packages/openbridge-webcomponents/src/navigation-instruments/readout-typography-issue/readout-typography-issue.css

```
@@ -0,0 +1,88 @@
+    var(
+      --instrument-components-readout-new-group-horizontal-enhanced-container-padding-horizontal
+    );
+}
+
+.content-container {
+  display: flex;
+  align-items: center;
+  column-gap: calc(
+    var(
+        --instrument-components-readout-new-group-horizontal-enhanced-value-container-gap
+      ) /
```

Stylelint (17.4.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)